Dockerfile.govuk-base

```diff
@@ -6,7 +6,7 @@
 FROM buildpack-deps
 
 # Install chrome and its dependencies
-RUN apt-get update -qq && apt-get install -y libxss1 libappindicator1 libindicator7
+RUN apt-get update -qq && apt-get install -y
```

Comment:
```suggestion
```

We can remove this line (same with the other Dockerfiles changed here) as it is no longer installing any software with apt